sdk/include/opentelemetry/sdk/metrics/aggregation/aggregation_config.h

```diff
@@ -13,13 +13,23 @@ namespace sdk
 {
 namespace metrics
 {
+
+enum class AggregationConfigType
+{
+  kDefault,
+  kHistogram,
+  kBase2ExponentialHistogram
+};
+
```

Comment:
Can we use the Aggression type from here - https://github.com/open-telemetry/opentelemetry-cpp/blob/b55766516d7acea524f6184103be691c996ac45e/sdk/include/opentelemetry/sdk/metrics/instruments.h#L44